mygithub/src/components/github/Profile.js

```diff
@@ -1,58 +1,59 @@
-import React,{Component} from 'react';
+import React, { Component } from 'react';
 import RepoList from './RepoList.js';
-class Profile extends Component{
-	
-	render(){
-// Nesting profile component in App component
-		return(
+class Profile extends Component {
+
+	render() {
+		// Nesting profile component in App component
+		return (
 			<div>
 				<div className="panel panel-default">
-					  <div className="panel-heading"></div>
-					  <div className="panel-body">
-					    <h1> Name :{this.props.userData.name} </h1>
-					  </div>
+					<div className="panel-heading"></div>
+					<div className="panel-body">
+						<h1> Name :{this.props.userData.name} </h1>
```

Comment:
Consider adding a check to handle cases where `this.props.userData.name`, `address`, etc. might be undefined. This will prevent errors and improve the robustness of the component. You can use optional chaining or a conditional statement for this purpose.